src/main/kotlin/io/github/inductiveautomation/kindling/thread/model/ThreadModel.kt

```diff
@@ -67,6 +67,21 @@ sealed class ThreadColumnList : ColumnList<ThreadLifespan>() {
         },
     )
 
+    val totalCPU = Column<ThreadLifespan, Double?>(
+        header = "Total CPU",
+        columnCustomization = {
+            minWidth = 60
+            cellRenderer = DefaultTableRenderer { value ->
+                (value as? Double)?.let { percent.format(it) }.orEmpty()
+            }
+        },
+        getValue = { threads ->
+            threads.sumOf { thread -> thread?.cpuUsage ?: 0.0 }
+        },
+    )
+
+
+
```

Comment:
Looks like these changes bled over from other work we did. This should be removed.